src/plugins/saved_objects_management/public/management_section/recent_work.tsx

```diff
@@ -232,9 +232,9 @@ export const RecentWork = (props: { core: CoreStart; workspaceEnabled?: boolean
                 recentAccessItem,
                 navLinks,
                 core.http.basePath,
-                core.application.navigateToUrl
+                core.application.navigateToUrl,
+                !!workspaceEnabled
```

Comment:
Shall we do a search for the places that are using `createRecentNavLink`? IIRC there are at least 2 places:
- recent menu in breadcrumb.
- recent works in home page.